motions/jumpToLink.ts

```diff
@@ -1,13 +1,22 @@
 import { jumpToPattern } from "../utils/jumpToPattern";
 import { MotionFn } from "../utils/vimApi";
 
-const WIKILINK_REGEX_STRING = "\\[\\[[^\\]\\]]+?\\]\\]";
-const MARKDOWN_LINK_REGEX_STRING = "\\[[^\\]]+?\\]\\([^)]+?\\)";
-const LINK_REGEX_STRING = `${WIKILINK_REGEX_STRING}|${MARKDOWN_LINK_REGEX_STRING}`;
+const WIKILINK_REGEX_STRING = "\\[\\[.*?\\]\\]";
```

Comment:
Also went ahead and improved the regexes here a little to be more concise and more accurate (e.g. I don't think the original `[^\\]\\]]` was actually doing what I wanted it to do in `WIKILINK_REGEX_STRING`)